src/components/SideMenu/styled.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+import { Close } from '@styled-icons/material-outlined/Close';
+
+const ContainerMenu = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.4);
+  transform: ${({ open }) => (open ? 'translateX(0)' : 'translateX(-100%)')};
```

Comment:
For not repeating code here or in future styles here or at other components where you could use same logic, this can be extracted in a named arrow fn like:
```js
const translateOnOpen = ({ open }) => (open ? 'translateX(0)' : 'translateX(-100%)');
```

Then here and a few lines below:
```suggestion
  transform: ${translateOnOpen};
```